### PR TITLE
Chapter 14: Following users

### PR DESCRIPTION
**Pull request by vuhuykhoi**
Screenshot from 2019-07-13 22-31-02

Chapter 14: Following users

### CODE REVIEW

**Comment by vuhuykhoi** [comment]
Screenshot from 2019-07-13 22-31-02

Chapter 14: Following users

**Comment by vuhuykhoi** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
```
Comment (hungcnh-0828) [inline app/models/user.rb]:
```suggestion
  has_many :active_relationships, class_name: Relationship.name,
```

File: app/models/user.rb [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+                                  foreign_key: "follower_id",
```
Comment (hungcnh-0828) [inline app/models/user.rb]:
```suggestion
                                  foreign_key: :follower_id,
```

**Pull request merged**